docs/content/integrations/duckdb/using-duckdb-with-dagster.mdx

```diff
@@ -5,16 +5,15 @@ description: Store your Dagster assets in DuckDB
 
 # Using Dagster with DuckDB
 
-This tutorial focuses on how to store and load Dagster's [Software-defined Assets (SDAs)](/concepts/assets/software-defined-assets) in DuckDB.
+This tutorial focuses on how to create and interact with DuckDB tables in Dagster's [Software-defined Assets (SDAs)](/concepts/assets/software-defined-assets).
 
-By the end of the tutorial, you will:
+The `dagster-duckdb` library provides two ways to interact with DuckDB tables. The [resource](TODO-LINK) allows you to directly run SQL queries against tables within an asset's compute function. The [I/O manager](TODO-LINK) transfers the responsibility of storing and loading DataFrames as DuckDB tables to Dagster.
```

Comment:
TODO - decide if the TODO links in this paragraph should link to the API docs for the resource and IO manager, or to the concept page for resources and io managers

Reply:
I feel like concept pages might be better.